packages/@aws-cdk-testing/framework-integ/test/aws-lambda/test/integ.runtime.inlinecode.ts

```diff
@@ -65,6 +65,13 @@ const node20xfn = new Function(stack, 'NODEJS_20_X', {
 });
 new CfnOutput(stack, 'NODEJS_20_X-functionName', { value: node20xfn.functionName });
 
+const node22xfn = new Function(stack, 'NODEJS_22_X', {
```

Comment:
later on we should add NodeJs function integ test cases to make sure that the build image is available